engine/class_modules/sc_death_knight.cpp

```diff
@@ -7782,15 +7783,18 @@ void death_knight_t::default_apl_frost()
   standard -> add_action( this, "Remorseless Winter", "", "Standard single-target rotation" );
   standard -> add_action( this, "Frost Strike", "if=cooldown.remorseless_winter.remains<=2*gcd&talent.gathering_storm.enabled" );
   standard -> add_action( this, "Howling Blast", "if=buff.rime.up" );
+  standard -> add_action( this, "Frostscythe", "if=talent.icecap.enabled&buff.killing_machine.up" );
+  standard -> add_action( this, "Obliterate", "if=talent.icecap.enabled&buff.pillar_of_frost.up&!azerite.latent_chill.enabled" );
   standard -> add_action( this, "Obliterate", "if=!buff.frozen_pulse.up&talent.frozen_pulse.enabled" );
   standard -> add_action( this, "Frost Strike", "if=runic_power.deficit<(15+talent.runic_attenuation.enabled*3)" );
   standard -> add_talent( this, "Frostscythe", "if=buff.killing_machine.up&rune.time_to_4>=gcd" );
   standard -> add_action( this, "Obliterate", "if=runic_power.deficit>(25+talent.runic_attenuation.enabled*3)" );
   standard -> add_action( this, "Frost Strike" );
   standard -> add_talent( this, "Horn of Winter" );
   standard -> add_action( "arcane_torrent" );
-
-  aoe -> add_action( this, "Remorseless Winter", "if=talent.gathering_storm.enabled|(azerite.frozen_tempest.rank&spell_targets.remorseless_winter>=3&!buff.rime.up)" );
+	
+  // AoE Rotation
```

Comment:
What's the point of this?
The additional "AoE Rotation" parameter outputs the "#AoE Rotation" comment in the default profiles.
The variable's name for this specific APL is already self explanatory so I don't think it warrants a c++ side code comment